Formula/gambit-scheme.rb

```diff
@@ -0,0 +1,28 @@
+class GambitScheme < Formula
+  desc "Gambit Scheme is an implementation of the Scheme Language"
+  homepage "http://gambitscheme.org"
+  url "https://github.com/gambit/gambit/archive/v4.9.1.tar.gz"
+  sha256 "667ae2ee657c22621a60b3eda6e242224d41853adb841e6ff9bc779f19921c18"
+
+  depends_on "openssl"
+
+  def install
+    args = %W[
+      --prefix=#{prefix}
+      --enable-single-host
+      --enable-multiple-versions
+      --enable-default-runtime-options=f8,-8,t8
```

Comment:
Is this the setting that upstream ships by default?

Reply:
It autoclosed due to push to my master.

> Follow up from #34503. Please do not open multiple PRs for a single formula, this messes up the review history.

Reply:
No they are not. As the defaults would not need to be articulated here as arguments right?
However this is the use case required by dependent systems.

Reply:
> the use case required by dependent systems

What dependent systems?

Reply:
These are the options required by Gerbil Scheme.

Reply:
These Options are the recommended performance options from the Gambit wiki, and the default runtime argument there adds uft8 support. Since the defaults assume a very minimal feature set.